mayastor/src/subsys/opts.rs

```diff
@@ -482,15 +482,36 @@ impl GetOpts for IscsiTgtOpts {
 pub struct ErrStoreOpts {
     /// ring buffer size
     pub err_store_size: usize,
+
     /// NexusErrStore enabled
     pub enable_err_store: bool,
+
+    /// whether to fault the child due to failed re-tried IOs
+    pub fault_child_on_retry_count: bool,
```

Comment:
I think it would be better to name this a particular action we can do, i.e retries_exceed = "action" where action is ingore, reset, fault, or whatever.

Reply:
Changed variable from a bool to an enum with currently 2 values, "fault" and "ignore"